doc/bucket-types.md

```diff
@@ -0,0 +1,41 @@
+[NooBaa Operator](../README.md) /
+
+# Bucket Types in NooBaa
+NooBaa supports two types of buckets - most commonly referred to as `namespace` and `data`. While all NooBaa buckets are accessible via the S3 API and appear the same to the user, they have different uses and implementations under the hood.
+This document will explaing the meaning of each, as well as the differences between the two.
+
+## Data Buckets
+Data buckets are the 'classic' type of bucket in NooBaa. In a classic NooBaa deployment, the product creates several resources by default - a backingstore (`noobaa-default-backing-store`), a bucketclass (`noobaa-default-bucket-class`) and a bucket (`first.bucket`) as part of the product's initialization process. When data is written to these buckets, it is first processed by NooBaa - the data is compressed, deduplicated, encrypted, and split into chunks. These chunks are then stored in the backingstore that the bucket is connected to.
+The only way to access the objects on data buckets is through NooBaa - the chunks cannot be used or deciphered without the system.
+
+## Namespace Buckets
+Namespace buckets work differently than data buckets, since they try¹ to not apply any processing on the objects that are uploaded to them, and act as more of a 'passthrough'. In cases where a non-S3 storage provider API is used, NooBaa takes care of bridging potential gaps between S3 and the provider's API - no user action is required. This also means that as long as NooBaa supports a certain provider's API, users can use the S3 API to manage it (while NooBaa takes care of the 'translation')
```

Comment:
If you want separation:
```suggestion
Namespace buckets work differently than data buckets, since they try¹ to not apply any processing on the objects that are uploaded to them, and act as more of a 'passthrough'. In cases where a non-S3 storage provider API is used, NooBaa takes care of bridging potential gaps between S3 and the provider's API - no user action is required. This also means that as long as NooBaa supports a certain provider's API, users can use the S3 API to manage it (while NooBaa takes care of the 'translation').  
```

Otherwise, it is in the same line...
![Screenshot 2024-11-26 at 9 59 34](https://github.com/user-attachments/assets/c96c1d16-39bb-4892-a36a-5df9199082d2)